docs/build-for-developers/cli-usage.md

```diff
@@ -1,98 +1,160 @@
-> file) For example `openfn execute hello.js ` Reads hello.js, looks for state
-> and output in foo
+**Auto-install adaptors and check if a matching version is found in the repo**
 
 ```bash
--i, --autoinstall Auto-install the language adaptor
--a, --adaptors, --adaptor A language adaptor to use for the job
+openfn path/to/job.js -ia adaptor-name
 ```
 
-If an adaptor is already installed by auto install, you can use the command
-without the `-i` options. i.e `openfn hello.js -a http`
+**Remove all adaptors from the repo**
 
-### Change log level
+```bash
+openfn repo clean
+```
 
-You can pass `-l info` or `--log info` to get more feedback about what's
-happening, or `--log debug` for more details than you could ever use. Below is
-the list of different log levels
+### Run adaptors from monorepo
+
+**Load from the monorepo using the `-m` flag**
 
 ```bash
-openfn hello.js -a http -l none
+openfn path/to/job.js -ma /path/to/monorepo
 ```
 
-| log level    | description                                              |
-| ------------ | -------------------------------------------------------- |
-| `-l none`    | Quiet mode                                               |
-| `-l default` | Top level information of what is happening               |
-| `-l info`    | Get more feedback on what is happening openfn            |
-| `-l debug`   | Get information about runtime, cli, compiler and the job |
+**Set the monorepo location using the `OPENFN_ADAPTORS_REPO` env var**
 
-### Compilation
+```bash
+export OPENFN_ADAPTORS_REPO=/path/to/monorepo
+```
 
-The CLI will attempt to compile your job code into normalized Javascript. It
-will do a number of things to make your code robust, portable, and easier to
-debug from a pure JS perspective.
+**Include `-m` to load from the monorepo**
 
 ```bash
-openfn compile [path]
+openfn path/to/job.js -ma adaptor-name
+```
+
+> Remember to build an adaptor before running.
+
+### Run a workflow
+
+<details>
+  <summary>A workflow has a structure like this</summary>
+
+```json
+{
+  "start": "a", // optionally specify the start node (defaults to jobs[0])
+  "jobs": [
+    {
+      "id": "a",
+      "expression": "fn((state) => state)", // code or a path
+      "adaptor": "@openfn/language-common@1.75", // specifiy the adaptor to use (version optional)
+      "data": {}, // optionally pre-populate the data object (this will be overriden by keys in in previous state)
+      "configuration": {}, // Use this to pass credentials
+      "next": {
+        // This object defines which jobs to call next
+        // All edges returning true will run
+        // If there are no next edges, the workflow will end
+        "b": true,
+        "c": {
+          "condition": "!state.error" // Not that this is an expression, not a function
+        }
+      }
+    }
+  ]
+}
 ```
 
-Will compile the openfn job and print or save the resulting js.
+</details>
+
+To run a workflow
+
+```bash
+openfn path/to/workflow.json -i
```

Comment:
Maybe it's time we autoinstall by default eh? Makes a lot of our documentation simpler

Reply:
@josephjclark  I was think the same thing but for local development a user must at-least setup their `repo` for local installation of adaptors. Then if i ran a job and forgotten `-i` it will check in the repo first then decide if we should download or not. 

It's a good developer experience but i think it might be a lot of work for `cli`